apps/mymedicare_cb/models.py

```diff
@@ -1,4 +1,87 @@
+import requests
+import logging
 from django.db import models
+from django.contrib.auth.models import User, Group
+from apps.accounts.models import UserProfile
+from apps.fhir.authentication import convert_sls_uuid
+from apps.fhir.bluebutton.models import Crosswalk
+from apps.fhir.bluebutton.utils import get_resourcerouter, FhirServerAuth
+
+__author__ = "Alan Viars"
```

Comment:
It looks like you copied this from the old versions of views.py, and I made a couple of changes in the last PR. This one, and the g_o_c below stick out, but I'm wondering whether there's anything more consequential